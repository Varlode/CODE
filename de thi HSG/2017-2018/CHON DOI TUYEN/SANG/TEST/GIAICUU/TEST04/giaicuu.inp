3
4 1
10
2
5
3
1 1 10
1 5
6 2 4 1 7
1 4 15
3 3
1 1 1
1 1 1
1 1 20
3 3 30