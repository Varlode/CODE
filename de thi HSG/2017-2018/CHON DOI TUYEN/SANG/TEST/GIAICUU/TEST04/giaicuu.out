YES
0
YES
2
YES
6